library/ceph_volume.py

```diff
@@ -646,7 +647,7 @@ def run_module():
             '--format=json',
         ]
 
-        cmd = batch(module, container_image)
+        cmd = batch(module, container_image, report=True)
```

Comment:
I don't think we want to hardcode `report=True` here, don't we want to use the value of `report` from a few lines up? Otherwise I think that every batch command will not include `--yes`.

Reply:
@andrewschoen no, this call is exclusively made for running a batch report.
Even though we don't call the module with `report: True` in ansible, this specific call always does a report. So it knows whether something is going to change before running the prepare command, if not we don't run the batch command. See L654-L657.

```python
L654: # Run batch --report to see what's going to happen
L655: # Do not run the batch command if there is nothing to do
L656: rc, cmd, out, err = exec_command(
L657:    module, batch_report_cmd)
```

Reply:
Ok, I see that now, thanks for the explanation. You also create an entirely new batch cmd on L697 and L702 so there's nothing to worry about here.